<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiveShareExportDialog</class>
 <widget class="QDialog" name="DiveShareExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frameConfigure">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>User ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtUID">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdClear">
          <property name="text">
           <string>⌫</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="getUIDbutton">
          <property name="text">
           <string>Get user ID</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#ff8000;&quot;&gt;⚠&lt;/span&gt; Not using a UserID means that you will need to manually keep bookmarks to your dives, to find them again.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkPrivate">
        <property name="toolTip">
         <string>Private dives will not appear in &quot;related dives&quot; lists, and will only be accessible if their URL is known.</string>
        </property>
        <property name="text">
         <string>Keep dives private</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="doUploadButton">
          <property name="text">
           <string>Upload dive data</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameResults">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextBrowser" name="txtResult">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
		<string></string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>68</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiveShareExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>getUIDbutton</sender>
   <signal>clicked()</signal>
   <receiver>DiveShareExportDialog</receiver>
   <slot>UIDFromBrowser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doUploadButton</sender>
   <signal>clicked()</signal>
   <receiver>DiveShareExportDialog</receiver>
   <slot>doUpload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdClear</sender>
   <signal>clicked()</signal>
   <receiver>txtUID</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiveShareExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getUID()</slot>
  <slot>doUpload()</slot>
 </slots>
</ui>
